actors/states/tree_top.go

```diff
@@ -0,0 +1,94 @@
+package states
+
+import (
+	"context"
+
+	address "github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/specs-actors/v2/actors/builtin"
+	init_ "github.com/filecoin-project/specs-actors/v2/actors/builtin/init"
+	"github.com/filecoin-project/specs-actors/v2/actors/util/adt"
```

Comment:
Within the migration package, I think every actors import should be explicitly versioned, e.g. `adt2`